src/realtime_helpers.cpp

```diff
@@ -115,13 +115,16 @@ bool lock_memory(std::string & message)
 #endif
 }
 
-std::pair<bool, std::string> set_thread_affinity(pthread_t thread, int core)
-{
-  std::string message;
 #ifdef _WIN32
-  message = "Thread affinity is not supported on Windows.";
+std::pair<bool, std::string> set_thread_affinity(HANDLE thread, int core)
```

Comment:
@SENAI-GilmarCorreia Don't you need a definition of this method in the header file?

Reply:
let me see If I can change this.

Reply:
No need. I'm already working on it now. I'll push it in few min